src/WebJobs.Extensions.DurableTask.Analyzers/AnalyzerReleases.Shipped.md

```diff
@@ -0,0 +1,30 @@
+﻿; Shipped analyzer releases
+; https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
+
```

Comment:
I think we need to add a Release header (e.g. `### Release 1.0.0`) to remove the warning that's showing up for line 4 in this file. Example: https://github.com/dotnet/roslyn-analyzers/blob/main/src/NetAnalyzers/Core/AnalyzerReleases.Shipped.md

Reply:
Done thanks